sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnarBatch.java

```diff
@@ -232,6 +233,56 @@ public MapData getMap(int ordinal) {
     public Object get(int ordinal, DataType dataType) {
       throw new NotImplementedException();
     }
+
+    @Override
+    public void setNullAt(int ordinal) {
+      columns[ordinal].putNull(rowId);
```

Comment:
hmm, good point. It'd be good safety measure, but might affect performance if do this additional check for each row/column.